src/pages/support-commands/details/index.css

```diff
@@ -0,0 +1,12 @@
+.form-card .icons {
```

Comment:
We should use tailwind css classes instead of plain css

Reply:
Used tailwind for .icons, but cannot use for the next svg, as it is a generic component. Will have to still use css here.